types/figma/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "dtslint/dt.json" ,
+    "rules": {
+        "dt-header": false,
+        "npm-naming": false,
+        "no-any-union": false,
```

Comment:
New packages should not override any lint rules. You explained why dt-header and npm-naming don't apply (thanks! it was a very clear explanation), but please remove the other overrides.